superset-frontend/plugins/plugin-chart-echarts/src/MixedTimeseries/EchartsMixedTimeseries.tsx

```diff
@@ -167,6 +171,10 @@ export default function EchartsMixedTimeseries({
               col: dimension,
               op: '==',
               val: values[i],
+              formattedVal:
+                DTTM_ALIAS === getColumnLabel(dimension)
+                  ? String(timestampFormatter(values[i]))
+                  : String(values[i]),
```

Comment:
same here, `DTTM_ALIAS` is already mostly deprecated, so we should refrain from using it in these types of checks. Again, this might be the correct solution for legacy charts that are powered by `viz.py`, but in the ECharts plugin we should not be using `DTTM_ALIAS` anymore.